pom.xml

```diff
@@ -4,7 +4,13 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.example</groupId>
+    <parent>
+        <artifactId>farao-parent-ws</artifactId>
+        <groupId>com.farao-community.farao</groupId>
+        <version>1.0.10</version>
+    </parent>
+
+    <groupId>com.farao-community.farao</groupId>
```

Comment:
Actually you can remove groupId when it is the same as pom parent

Reply:
true!